server/auth/authrequest.js

```diff
@@ -54,9 +54,12 @@ function performRequest(endpoint, method, data, fiwareHeaders) {
       resolveWithFullResponse: true,
       json: true // Automatically stringifies the body to JSON
     };
+    if (typeof payload !== 'undefined') {
+      endpoint += '?' + querystring.stringify(payload);
+      options.uri = host + ':' + port + endpoint;
```

Comment:
then name it better to understand it is an option not a payload and in all case it is a queryString not a payload